source/type/vehicle_journey.cpp

```diff
@@ -162,6 +162,11 @@ uint32_t VehicleJourney::earliest_time() const {
     return std::min(st.arrival_time, st.boarding_time);
 }
 
+uint32_t VehicleJourney::last_time() const {
+    const auto& st = navitia::last_stop_time(stop_time_list);
```

Comment:
why not just return stop_time_list.back() ? The stop_time_list may not be sorted by increasing times ?

Reply:
Since 'earliest_stop_time' exists, i think stop_time_list probably not ordered.